refinery/tool_manager/models.py

```diff
@@ -202,32 +197,14 @@ def get_extra_directories(self):
                 "Workflow-based tools don't utilize `extra_directories`"
             )
 
-    def get_workflow(self):
-        """
-        Fetch Workflow object for a Workflow-based ToolDefinition
-
-        :return: <Workflow>
-        :raises: RuntimeError, NotImplementedError
-        """
-        if self.tool_type == ToolDefinition.VISUALIZATION:
-            raise NotImplementedError(
-                "Visualization-based Tools don't utilize Workflows"
-            )
-        try:
-            return Workflow.objects.get(
-                internal_id=self.galaxy_workflow_id,
-                workflow_engine=self.workflow_engine
-            )
-        except(Workflow.DoesNotExist,
-               Workflow.MultipleObjectsReturned) as e:
-            raise RuntimeError("Couldn't fetch Workflow: {}".format(e))
-
 
 @receiver(pre_delete, sender=ToolDefinition)
-def delete_parameters_and_output_files(sender, instance, *args, **kwargs):
+def remove_associated_objects(sender, instance, *args, **kwargs):
```

Comment:
Is there a consistent practice around `delete_` vs `remove_`? Just noticing that the next method down is still `delete_*`.